internal/spentenergy/spentenergy.go

```diff
@@ -13,17 +14,51 @@ const (
 )
 
 func WalkingSpentCalories(steps int, weight, height float64, duration time.Duration) (float64, error) {
-	// TODO: реализовать функцию
+	if steps <= 0 {
+		return 0, fmt.Errorf("invalid argument steps = %v", steps)
```

Comment:
сообщение не передает суть проблемы
```suggestion
		return 0, fmt.Errorf("steps must be greater than zero, steps = %v", steps)
```